src/lib/drupal/query.ts

```diff
@@ -135,6 +135,7 @@ export const entityBaseFields = (entity: NodeTypes): PublishedEntity => {
     title: entity.title,
     metatags: entity.metatag,
     breadcrumbs: entity.breadcrumbs,
-    lastUpdated: entity.field_last_saved_by_an_editor || entity.created,
+    lastUpdated:
+      entity.field_last_saved_by_an_editor || entity.changed || entity.created,
```

Comment:
Slack thread for context: https://dsva.slack.com/archives/C07B86KV4P4/p1729096713224239